Makefile

```diff
@@ -1,6 +1,6 @@
 PD_PKG := github.com/tikv/pd
 
-TEST_PKGS := $(shell find . -iname "*_test.go" -exec dirname {} \; | \
+TEST_PKGS := $(shell find . ! -path "./tests/*" -iname "*_test.go" -exec dirname {} \; | \
```

Comment:
So the `BASIC_TEST_PKGS` below is unnecessary now?

Reply:
github set TEST_PKGS by "o list ./... | grep -v -E "github.com/tikv/pd/server/api|github.com/tikv/pd/tests/client|github.com/tikv/pd/tests/server/tso" > packages.list"